### PR TITLE
Python-Plugin-upgrade changes

### PR DESCRIPTION
**Pull request by prashelke**

### CODE REVIEW

File: GingerPythonPluginConsole/Ginger.PluginPackage.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
   "Id": "Python",
-  "Version": "v1.2",
+  "Version": "v2.1",
```
Comment (Maheshkale447) [inline GingerPythonPluginConsole/Ginger.PluginPackage.json]: change it to v2.0.0

File: GingerPythonPluginConsole/Properties/PublishProfiles/FolderProfile1.pubxml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
```
Comment (Maheshkale447) [inline GingerPythonPluginConsole/Properties/PublishProfiles/FolderProfile1.pubxml]: remove unwanted files

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: GingerPythonPluginConsole/Properties/PublishProfiles/FolderProfile1.pubxml (hunk 2 above)
Comment (prashelke) [reply GingerPythonPluginConsole/Properties/PublishProfiles/FolderProfile1.pubxml]: Done

File: GingerPythonPluginConsole/Ginger.PluginPackage.json (hunk 1 above)
Comment (prashelke) [reply GingerPythonPluginConsole/Ginger.PluginPackage.json]: Done

**Review by Maheshkale447 — APPROVED** [review]

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**